floor/lib/src/callback.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:sqflite/sqflite.dart' as sqflite;
```

Comment:
Library prefix `as sqflite` can be removed as there are no naming collisions in this class.